### PR TITLE
Export LocalKeyVault

### PR DESCRIPTION
**Pull request by cosminm-affnd**
Export LocalKeyVault so it can be used by other libraries.

### CODE REVIEW

**Review by dendavidov-affinidi — DISMISSED** [review]

File: common-libs/common/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# 2.0.0-beta.5 (2020-08-23)
```
Comment (DenisPopov15) [inline common-libs/common/CHANGELOG.md:1]: @cosminm-affnd should be `2.0.0-beta.6` here ?

**Review by DenisPopov15 — DISMISSED** [review]

File: common-libs/common/CHANGELOG.md (hunk 1 above)
Comment (cosminm-affnd) [reply common-libs/common/CHANGELOG.md]: oups, updated

**Review by DenisPopov15 — APPROVED** [review]

File: common-libs/common/src/services/DidDocumentService/KeyVault.ts [hunk 2]
```diff
@@ -23,4 +23,11 @@ export interface KeyVault {
    * @param payload payload to be signed
    */
   sign(payload: Buffer): Buffer
+
+  /**
+   * Sign the given payload with the primary public key. Same as sign but async.
+   *
+   * @param payload payload to be signed
+   */
+  signAsync?(payload: Buffer): Promise<Buffer>
```
Comment (Yengas) [inline common-libs/common/src/services/DidDocumentService/KeyVault.ts:32]: I did not get how this helps me. I still need to implement the sync version of this function right. No one uses the `signAsync` as well? 🤔

Comment (cosminm-affnd) [reply common-libs/common/src/services/DidDocumentService/KeyVault.ts:32]: The issue I had was when implementing a KeyVault backed by metamask, the sign method there is async, and adding an optional async method here was the only work around I could find. Other approaches I could use?

**Review by Yengas — APPROVED** [review]

**Pull request merged**